dbt/adapters/spark/impl.py

```diff
@@ -400,13 +383,120 @@ def run_sql_for_tests(self, sql, fetch, conn):
         finally:
             conn.transaction_open = False
 
+    @available.parse_none
+    def submit_python_job(self, parsed_model:dict, compiled_code: str, timeout=None):
```

Comment:
The implementation should be in `dbt-databricks`.

Reply:
Btw, will this have a default implementation raising `NotImplemented` or something in `dbt-core`?